lib/css/components/_stacks-navigation.less

```
@@ -50,6 +51,24 @@
             background: var(--theme-primary-color);
             color: var(--white);
         }
+
+        &.s-navigation--item__dropdown {
+            padding-right: 2em;
+
+            &:after {
+                content: "";
+                position: absolute;
+                z-index: @zi-active;
+                top: calc(50% - 2px);
+                right: 0.9em;
```

I notice button takes the same approach to positioning a caret. Do you know of any reason we're going with absolute? We _could_ do something like this:

```suggestion
            &:after {
                content: "";
                margin-left: 0.5em;
```

This is small potatoes and I consider "consistency" and "it works fine" a valid reason, so feel free to ignore this suggestion if that's the case.

---

I think this was to control the vertical alignment to the pixel level. Happy to revisit this!